var/spack/repos/builtin/packages/pumi/package.py

```diff
@@ -109,15 +109,15 @@ def test(self):
         if self.spec.version <= Version("2.2.6"):
             return
         exe = "uniform"
-        options = ["../testdata/pipe.dmg", "../testdata/pipe.smb", "pipe_unif.smb"]
+        options = ["../share/testdata/pipe.dmg", "../share/testdata/pipe.smb", "pipe_unif.smb"]
         expected = "mesh pipe_unif.smb written"
         description = "testing pumi uniform mesh refinement"
         self.run_test(exe, options, expected, purpose=description, work_dir=self.prefix.bin)
 
         mpiexec = Executable(join_path(self.spec["mpi"].prefix.bin, "mpiexec")).command
         mpiopt = ["-n", "2"]
         exe = ["split"]
-        options = ["../testdata/pipe.dmg", "../testdata/pipe.smb", "pipe_2_.smb", "2"]
+        options = ["../share/testdata/pipe.dmg", "../share/testdata/pipe.smb", "pipe_2_.smb", "2"]
```

Comment:
You can provide full paths here:

```suggestion
        options = [
            join_path(data_dir, "pipe.dmg"),
            join_path(data_dir, "pipe.smb"),
            "pipe_2_.smb",
            "2",
        ]
```